akka-http2-support/build.sbt

```diff
@@ -1,13 +1,36 @@
 import akka._
+import java.nio.file.Files
+import java.nio.file.attribute.{ PosixFileAttributeView, PosixFilePermission }
 
 Dependencies.http2
+Dependencies.http2Support
 
 fork in run in Test := true
 fork in Test := true
 
 connectInput in run in Test := true
 
-javaOptions in run in Test += "-javaagent:jetty-alpn-agent-2.0.5.jar"
+javaOptions in Test += "-Dh2spec.path=" + ((target in Test).value / Dependencies.h2SpecPackageName / s"h2spec${exeIfWindows}")
 
-javaOptions in test in Test += "-javaagent:jetty-alpn-agent-2.0.5.jar"
-javaOptions in testOnly in Test += "-javaagent:jetty-alpn-agent-2.0.5.jar"
+def exeIfWindows = {
+  val os = System.getProperty("os.name").toLowerCase()
+  if (os startsWith "win") ".exe"
+  else ""
+}
+
+resourceGenerators in Test += Def.task {
+  val h2SpecOut = (target in Test).value
+  streams.value.log.info("Extracting h2spec binary")
```

Comment:
This will always unzip the h2spec distribution, however, it looks fast enough to not matter. It would be nice however to figure out a way to put the raw binary into the Ivy cache so this resource generator would not be necessary.